python/paddle/fluid/tests/unittests/test_conv2d_mkldnn_op.py

```diff
@@ -20,16 +20,19 @@
 class TestMKLDNN(TestConv2dOp):
     def init_kernel_type(self):
         self.use_mkldnn = True
+        self.data_format = "NCHW"
 
 
 class TestMKLDNNWithPad(TestWithPad):
     def init_kernel_type(self):
         self.use_mkldnn = True
+        self.data_format = "NCHW"
 
 
 class TestMKLDNNWithStride(TestWithStride):
     def init_kernel_type(self):
         self.use_mkldnn = True
+        self.data_format = "NCHW"
```

Comment:
Can you tell me why the default format is `NCHW` ?

Reply:
Because the tests were written assuming NCHW layout. MKLDNN used NCHW previously and it worked. If you change it to NHWC it doesn't work.